<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BastionWindow</class>
 <widget class="QMainWindow" name="BastionWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1045</width>
    <height>775</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
color: white;
background-color: rgb(50, 50, 50);
selection-color: white;
selection-background-color:: rgb(208,206,44)
}

QWidget:disabled{
color: gray;
}


QLineEdit:focus,QSpinBox:focus,QDoubleSpinBox:focus,QComboBox:focus,QListView:focus
{
    border: 2px solid rgb(208,206,44);
}

/*
QWidget:selected{
border-width: 2px;
border-color: rgb(208,206,44);
border-style: solid;
border-radius: 3;
}
*/

QPushButton, QLineEdit, QComboBox, QSpinBox,QDoubleSpinBox,QListView{
color: white;
background-color: rgb(80, 80, 80);
selection-color: white;
selection-background-color:: rgb(208,206,44)
}

QPushButton, QLineEdit, QComboBox, QSpinBox ,QDoubleSpinBox, QTabWidget::pane,QListView,QGroupBox{
border-width: 1px;
border-color: black;
border-style: solid;
border-radius: 3;
}


QDockWidget::title {
  font: bold;
  text-align: center;
  background: rgb(90, 90, 90);
  border-width: 1px;
  border-color: black;
  border-style: solid;
  border-radius: 1;
}

QDockWidget{
border-width: 1px;
border-color: black;
border-style: solid;
border-radius: 3;
}

QPushButton {
padding: 3px;
font-size: 10px;
padding-left: 5px;
padding-right: 5px;
min-width: 50px;
max-width: 50px;
min-height: 13px;
max-height: 13px;
}

QPushButton:hover {
  background-color: rgb(100,100, 100)
}

QPushButton:pressed {
  background-color: rgb(208,206,44)
}

QLabel ,QCheckBox{
    color: white;
}


QAbstractButton:disabled {
    color:rgb(100, 100, 100) ;
}

QLineEdit  {
background-color: rgb(70, 70, 70);
selection-background-color: rgb(208,206,44)
}


 QTabWidget::tab-bar  {
        left: 5px; 
    }

    QTabBar::tab  {
    color: white;
    border: 1px solid black;
    border-bottom-color:rgb(50,50,50); 
    border-top-left-radius: 3px;
    border-top-right-radius: 3px;
    padding: 3px;
font-size: 12px;
padding-left: 5px;
padding-right: 5px;
}
  
    QTabBar::tab:!selected  {
    background-color: rgb(25, 25, 25);
        margin-top: 2px; 
    }




QComboBox {
padding: 3px;
padding-left: 5px;
padding-right: 5px;
min-width: 50px;
min-height: 13px;
}
     

QCheckBox,QGroupBox{
    spacing: 5px;
}
 
QCheckBox::indicator,QGroupBox::indicator,QListWidget::indicator{
    width: 16px;
    height: 16px;
}
 
QCheckBox::indicator:unchecked,QGroupBox::indicator:unchecked,QListWidget::indicator:unchecked {
    image: url(:/images/checkbox_unchecked.png);
}

QCheckBox::indicator:checked:disabled ,QGroupBox::indicator:checked:disabled,QListWidget::indicator:disabled {
    image: url(:/images/checkbox_checked_noSelect.png);
}
 
QCheckBox::indicator:checked ,QGroupBox::indicator::checked,QListWidget::indicator:checked {
    image: url(:/images/checkbox_checked.png);
}

QGroupBox {
padding-top: 10px;
}

QMenu
{
    border: 1px solid #000;
}

QMenu::item
{
    padding: 2px 20px 2px 20px;
}

QMenu::item:selected
{
    color: rgb(208,206,44);
}

QComboBox
{
    selection-background-color: #ffaa00;
    background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #565656, stop: 0.1 #525252, stop: 0.5 #4e4e4e, stop: 0.9 #4a4a4a, stop: 1 #464646);
    border-style: solid;
    border: 1px solid #1e1e1e;
    border-radius: 5;
}

QComboBox:on
{
    padding-top: 3px;
    padding-left: 4px;
    background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #2d2d2d, stop: 0.1 #2b2b2b, stop: 0.5 #292929, stop: 0.9 #282828, stop: 1 #252525);
}
QComboBox:disabled
{
    background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #2d2d2d, stop: 0.1 #2b2b2b, stop: 0.5 #292929, stop: 0.9 #282828, stop: 1 #252525);
}



QComboBox,QAbstractItemView
{
    border-width: 1px;
    border-color: black;
    border-style: solid;
    border-radius: 3;;
    selection-background-color:rgb(208,206,44);
}

QComboBox::drop-down
{
     subcontrol-origin: padding;
     subcontrol-position: top right;
     width: 15px;

     border-left-width: 1px;
     border-left-color: black;
     border-left-style: solid; /* just a single line */
     border-top-right-radius: 3px; /* same radius as the QComboBox */
     border-bottom-right-radius: 3px;
 }

QComboBox::down-arrow
{
     image: url(:/images/down_arrow.png);
}

QSpinBox  {
    padding-right: 15px; /* make room for the arrows */
selection-background-color:: rgb(208,206,44)
}
 

QSpinBox::up-button  {
    subcontrol-origin: border;
    subcontrol-position: top right; /* position at the top right corner */
    width: 16px; /* 16 + 2*1px border-width = 15px padding + 3px parent border */
    border-width: 1px;
}
 
 
QSpinBox::up-arrow  {
    image: url(:/images/up_arrow.png);
    width: 7px;
    height: 7px;
}

QSpinBox::up-arrow::pressed  {
      image: url(:/images/up_arrow_dis.png);
    width: 6px;
    height: 6px;
}

 

QSpinBox::up-arrow:disabled, QSpinBox::up-arrow:off  { /* off state when value is max */
   image: url(:/images/up_arrow_dis.png);
}
 
QSpinBox::down-button  {
    subcontrol-origin: border;
    subcontrol-position: bottom right; /* position at bottom right corner */
    width: 16px;
    border-width: 1px;
    border-top-width:0;
}
 
 
QSpinBox::down-arrow  {
    image: url(:/images/down_arrow.png);
    width: 7px;
    height: 7px;
}

QSpinBox::down-arrow::pressed  {
      image: url(:/images/down_arrow_dis.png);
    width: 6px;
    height: 6px;
}
 
QSpinBox::down-arrow:disabled,
QSpinBox::down-arrow:off  { /* off state when value in min */
   image: url(:/images/down_arrow_dis.png);
}
 

QScrollBar:horizontal {
     border: 1px solid #222222;
     background-color: rgb(50, 50, 50);
     height: 9px;
     margin: 0px 16px 0 16px;
}

QScrollBar::handle:horizontal
{
      background: rgb(223,223,223);
      min-height: 20px;
      border-radius: 2px;
}

QScrollBar::add-line:horizontal {
      border: 1px solid black;
      border-radius: 2px;
      background: rgb(223,223,223);
      width: 14px;
      subcontrol-position: right;
      subcontrol-origin: margin;
}

QScrollBar::sub-line:horizontal {
      border: 1px solid black;
      border-radius: 2px;
      background: rgb(223,223,223);
      width: 14px;
     subcontrol-position: left;
     subcontrol-origin: margin;
}

QScrollBar::right-arrow:horizontal, QScrollBar::left-arrow:horizontal
{
      border: 1px solid black;
      width: 1px;
      height: 1px;
      background: white;
}

QScrollBar::add-page:horizontal, QScrollBar::sub-page:horizontal
{
      background: none;
}

QScrollBar:vertical
{
      background-color: rgb(50, 50, 50);
      width: 12px;
      margin: 16px 0 16px 0;
      border: 1px solid #222222;
}

QScrollBar::handle:vertical
{
      background: rgb(223,223,223);
      min-height: 20px;
      border-radius: 2px;
}

QScrollBar::add-line:vertical
{
      border: 1px solid black;
      border-radius: 2px;
      background: rgb(223,223,223);
      height: 14px;
      subcontrol-position: bottom;
      subcontrol-origin: margin;
}

QScrollBar::sub-line:vertical
{
      border: 1px solid black;
      border-radius: 2px;
      background: rgb(223,223,223);
      height: 14px;
      subcontrol-position: top;
      subcontrol-origin: margin;
}

QScrollBar::up-arrow:vertical, QScrollBar::down-arrow:vertical
{
      border: 1px solid black;
      width: 1px;
      height: 1px;
      background: white;
}

QHeaderView::section 
{
    background-color:  rgb(35,35,35);
    border: 0px solid #6c6c6c;
}

QTableWidget {
    border: 1px solid black;
    alternate-background-color:  rgb(45,45,45);
    background-color:  rgb(25,25,25);
}

QTableWidget::item:selected{
    background-color: rgb(208,206,44);
    color: rgb(10, 10, 10);
}

QTreeWidget {
    alternate-background-color:  rgb(35,35,35);
}


QTreeWidget::item:selected{
    background-color: rgb(208,206,44);
    color: rgb(10, 10, 10);
}

QTreeWidget::branch:has-siblings:!adjoins-item {
    border-image: url(:images/vline.png) 0;
}

QTreeWidget::branch:has-siblings:adjoins-item {
    border-image: url(:images/branch-more.png) 0;
}

QTreeWidget::branch:!has-children:!has-siblings:adjoins-item {
    border-image: url(:images/branch-end.png) 0;
}

QTreeWidget::branch:has-children:!has-siblings:closed,
QTreeWidget::branch:closed:has-children:has-siblings {
        border-image: none;
        image: url(:images/branch-closed.png);
}

QTreeWidget::branch:open:has-children:!has-siblings,
QTreeWidget::branch:open:has-children:has-siblings  {
        border-image: none;
        image: url(:images/branch-open.png);
}


QDoubleSpinBox  {
    padding-right: 15px; /* make room for the arrows */
selection-background-color:: rgb(208,206,44)
}
 

QDoubleSpinBox::up-button  {
    subcontrol-origin: border;
    subcontrol-position: top right; /* position at the top right corner */
    width: 16px; /* 16 + 2*1px border-width = 15px padding + 3px parent border */
    border-width: 1px;
}
 
 
QDoubleSpinBox::up-arrow  {
    image: url(:/images/up_arrow.png);
    width: 7px;
    height: 7px;
}

QDoubleSpinBox::up-arrow::pressed  {
      image: url(:/images/up_arrow_dis.png);
    width: 6px;
    height: 6px;
}

 

QDoubleSpinBox::up-arrow:disabled, QDoubleSpinBox::up-arrow:off  { /* off state when value is max */
   image: url(:/images/up_arrow_dis.png);
}
 
QDoubleSpinBox::down-button  {
    subcontrol-origin: border;
    subcontrol-position: bottom right; /* position at bottom right corner */
    width: 16px;
    border-width: 1px;
    border-top-width:0;
}
 
 
QDoubleSpinBox::down-arrow  {
    image: url(:/images/down_arrow.png);
    width: 7px;
    height: 7px;
}

QDoubleSpinBox::down-arrow::pressed  {
      image: url(:/images/down_arrow_dis.png);
    width: 6px;
    height: 6px;
}
 
QDoubleSpinBox::down-arrow:disabled,
QDoubleSpinBox::down-arrow:off  { /* off state when value in min */
   image: url(:/images/down_arrow_dis.png);
}

QTextEdit{
color: white;
background-color: rgb(0, 0, 0);
}

QListView {
    border: 1px solid black;
    alternate-background-color:  rgb(45,45,45);
    background-color:  rgb(25,25,25);
}

QListView::item:selected:active {
     background:  rgb(208,206,44);
 }

QListView::disabled {
   background:  rgb(50,50,50);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1045</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuAdd">
    <property name="title">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuMisc">
    <property name="title">
     <string>Misc</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuAdd"/>
   <addaction name="menuMisc"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExit">
   <property name="text">
    <string>Save Script</string>
   </property>
  </action>
  <action name="actionSpool_Help">
   <property name="text">
    <string>Spool Help</string>
   </property>
  </action>
  <action name="actionSave_Script">
   <property name="text">
    <string>Save Script</string>
   </property>
  </action>
  <action name="actionExit_2">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
